### PR TITLE
feat: Add support for computed_members_map variable

### PR DESCRIPTION
**Pull request by zied-elouaer**

### CODE REVIEW

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -274,7 +283,71 @@ section {
...
+          required    = true
+          type        = string
+          description = <<-END
+            The role that should be applied.
+          END
+        }
+
+        attribute "members" {
+          type        = set(string)
+          default     = var.members
```
Comment (mariux) [inline README.tfdoc.hcl:310]: that is not true anymore... ;) 

i think it is required now?

**Review by mariux — CHANGES_REQUESTED** [review]

File: README.tfdoc.hcl (hunk 1 above)
Comment (zied-elouaer) [reply README.tfdoc.hcl]: Nice catch :) Fixed.

**Review by waxb — COMMENTED** [review]
lgtm, waiting for @waxb to release https://github.com/mineiros-io/terraform-google-pubsub-subscription/commit/b0ac58b004dd25c57696561c7ea689e5625c6d28

**Review by waxb — APPROVED** [review]
lgtm

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**